src/server/server.ts

```diff
@@ -1,16 +1,16 @@
 import express from 'express';
-import bodyParser from 'body-parser';
+//import bodyParser from 'body-parser';
 import swaggerui from '../swaggerRouter';
-import exampleApi from '../api/getEnrichments';
+//import exampleApi from '../api/getEnrichments';
 
 const API_PREFIX = '/api/v1';
 
 const app = express();
 
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
+//app.use(bodyParser.json());
+//app.use(bodyParser.urlencoded({ extended: true }));
 
-app.use(API_PREFIX, exampleApi);
+//app.use(API_PREFIX, exampleApi);
```

Comment:
You can perform those changes in your repository, but I dont want it to be in the pull request.
All the code in the pull request should be code that will eventually run.
If you comment out those lines it will break the application.
So you can leave the comments in your repository if you want, but you need to remove those changes from the pull request